scripts/filter_by_definition_or_context.py

```diff
@@ -0,0 +1,19 @@
+import jsonlines
+
+data_file = 'data/ontoemma.context.test'
```

Comment:
If this script is likely to be used for further experiments, consider adding in the readme file an example invocation of this script, a brief description of what it does and why, and how to get the input file.